src/app/component/MissionCard.jsx

```diff
@@ -76,13 +90,72 @@ const MissionCard = withStyles(styles)(({ children, classes, mission, ...rest })
 
   return (
     <Card className={classes.root} {...rest}>
-      <CardHeader
-        title={title}
-        titleTypographyProps={{ variant: "h2", component: "span" }}
-        className={classes.cardHeader}
-      />
       <CardContent className={classes.cardContent}>
-        <MissionCardContent contentItems={contentItems} classes={classes} />
+        <Grid container spacing="1" alignItems="flex-start" justify="flex-end" direction="row">
+          <Grid item>
+            <img height="20" src={appleIcon} alt="" />
+          </Grid>
+          <Grid item style={{ flex: 1 }}>
+            {title}
+          </Grid>
+          <Grid item>
+            <InfoIcon />
+          </Grid>
+        </Grid>
+      </CardContent>
+      <CardContent className={classes.cardContent}>
+        <Grid container direction="row">
+          <Grid item className={classes.halfRow}>
+            <Grid container direction="column">
+              <Grid item>
+                <Grid container direction="row">
+                  <Grid item>
+                    <PublishIcon />
+                  </Grid>
+                  <Grid item>
+                    <Typography variant="h5" noWrap>
+                      PICK UP
```

Comment:
It'd be great if we could make these be green / purples per the design - just so they fade into the background a bit more I gueess.